x.ps1

```diff
@@ -7,57 +7,58 @@ $ErrorActionPreference = "Stop"
 # syntax check
 Get-Command -syntax ${PSCommandPath} >$null
 
-$xpy = Join-Path $PSScriptRoot x.py
-# Start-Process for some reason splits arguments on spaces. (Isn't powershell supposed to be simpler than bash?)
-# Double-quote all the arguments so it doesn't do that.
-$xpy_args = @("""$xpy""")
+$pythonScriptArgs = @("$($PSScriptRoot)\x.py")
```

Comment:
Does that work when there's a space in the value of `$PSScriptRoot`?
And why did you remove the comment?